rb-files/check.rb

```diff
@@ -0,0 +1,9 @@
+require './persons'
+require './decorator'
```

Comment:
- [ ] I couldn't access your app because you did not apply the suggestion pointed out by the previous reviewer. Please kindly use `require_relative` in order to access files in this directory. ℹ️ 
I got this 👇 

<img width="758" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/231160689-2feb0e29-c83d-415e-b722-6a8c4c365948.png">

```suggestion
require_relative './persons'
require_relative './decorator'
```
<img width="588" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/231160341-4da36b35-245b-4fe7-bec7-ae06fe510561.png">